cli.js

```diff
@@ -3,14 +3,23 @@
 const meow = require('meow');
 const createInkApp = require('.');
 
-meow(`
+const cli = meow(`
 	Usage
 		$ mkdir my-cli
 		$ cd my-cli
-	  $ create-ink-app
+	  $ create-ink-app <template>
+
+	Flags
+		--help
+
+	Commands
+		create-ink-app <template>
```

Comment:
Instead of accepting an argument, could you add a `--typescript, --ts` flag to use TypeScript template instead?

Reply:
Of course! Flags for template selection feel more suited